### DISCUSSION TITLE
Making audio input a indiv. output

### DISCUSSION THREAD

**Discussion by cyberlazi** [bangcorrupt/hacktribe#328] [Ideas]
Hey i struggle a bit with the internal mfx on the electribe and the fact that it only has 2 main outputs.
i thought could the "audio input" be transformed into an individual output like the korg ESX has?

**Comment by bangcorrupt** [comment]
This can't be done in software, but we may be able to add more outputs with some hardware mods.  The SPORT peripheral on the DSP supports TDM mode, so it may be possible to use a multi-channel DAC.
